install_executorch.py

```diff
@@ -180,9 +181,13 @@ def main(args):
     # This option is used in CI to make sure that PyTorch build from the pinned commit
     # is used instead of nightly. CI jobs wouldn't be able to catch regression from the
     # latest PT commit otherwise
-    install_requirements(use_pytorch_nightly=not args.use_pt_pinned_commit)
-    os.execvp(
```

Comment:
I had to change back from  os.execvp to subprocess since install_optional_example_requirement is never reachable because os.execvp completely replaces the current process with the pip install command